### DISCUSSION TITLE
Implement a UI mechanism to delete saved variable option, no need to edit the raw JSON document

### DISCUSSION THREAD

**Discussion by wengelbrecht-grafana** [grafana/grafana#43959] [Dashboards, Panels & Visualizations]
In Grafana, you can pick your template variables, and when you save the dashboard have the option to save these as defaults:
![image](https://user-images.githubusercontent.com/83230950/149146678-88e647d6-8b2d-4dd7-b63e-2ff75f8c06da.png)

However, once you have a saved variable, it is not possible to tell Grafana to remove this saved variable as the default, through the UI. You have to go to the JSON document of the Dashboard and delete the saved variable in there. This is not really ideal as you need to understand JSON, where to find it in the JSON document and hope you didn't forget to have the right closing } or , in there. 

The use case I am working on, we have a list (generated) of calendar days that a user can select: 
![image](https://user-images.githubusercontent.com/83230950/149146963-a2098758-7948-450c-9908-a894c1aad01d.png)

Accidentally, the dashboard was saved with 2022-01-05 as the default. Now we need to save the dashboard every day with "today's" value as the default so that it is easy for all the other users to use this dashboard. But to remove this default save state, we need to edit and remove the following section from the JSON by hand: 
![image](https://user-images.githubusercontent.com/83230950/149147274-a6565520-4c35-4e7f-8bd2-916bca627f1a.png)

It would be nice to have a feature in the UI where you can go and select your saved variables and unsave them if that makes sense. Perhaps a tickbox on the page where template variables are defined already. It could show you the current default for this variable, and you can click a box or something to unsave it as the default. No need to edit the JSON document by hand.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!

**Comment by miguelcb84** [comment]
I support this feature, I've faced the same issue many times
